src/components/users/MobileStyle.css

```diff
@@ -0,0 +1,310 @@
+@media(max-width: 375px){
+
+    /* ---HOME PAGE --- */
+   
+    div#root {
+       
+        width: 100%;
+   
+    }
+
+    div.ui.large.fluid.pointing.vertical.tabular.menu {
+        display:flex;
+        flex-direction:row;
+        margin-top:520px;
+
+        /* overflow:hidden; */
+        background: lightgrey;
+        position: fixed;
+        z-index: 1;
+        margin-left: -3.6%;
```

Comment:
Rather than doing `margin-left: -3.6%` and `margin-top: 520px` try `left: 0` and `bottom: 0`

Reply:
Thank you. I kept trying to do margin-left:0 and margin-bottom:0 and that wouldn't work. Thank you so much. You're a rock star!!!